contracts/BIFI/strategies/Pangolin/StrategyPangolinMiniChefLP.sol

```diff
@@ -266,13 +293,37 @@ contract StrategyPangolinMiniChefLP is StratManager, FeeManager {
 
         IERC20(lpToken1).safeApprove(unirouter, 0);
         IERC20(lpToken1).safeApprove(unirouter, type(uint256).max);
+
+        if (rewardToOutputRoute[0] != nullAddress) {
+                IERC20(rewardToOutputRoute[0]).safeApprove(unirouter, 0);
+                IERC20(rewardToOutputRoute[0]).safeApprove(unirouter, uint256(-1));
+        }
     }
 
     function _removeAllowances() internal {
         IERC20(want).safeApprove(chef, 0);
         IERC20(output).safeApprove(unirouter, 0);
         IERC20(lpToken0).safeApprove(unirouter, 0);
         IERC20(lpToken1).safeApprove(unirouter, 0);
+
+        if (rewardToOutputRoute[0] != nullAddress) {
+            IERC20(rewardToOutputRoute[0]).safeApprove(unirouter, 0);
+        }
+    }
+
+    function setRewardRoute(address[] memory _rewardToOutputRoute) external onlyOwner {
+        require(_rewardToOutputRoute[_rewardToOutputRoute.length - 1] == output, "rewardToOutputRoute[last] != output");
+        rewardToOutputRoute = _rewardToOutputRoute;
+        IERC20(_rewardToOutputRoute[0]).safeApprove(unirouter, 0);
+        IERC20(_rewardToOutputRoute[0]).safeApprove(unirouter, uint256(-1));
+    }
+
+    function removeRewardRoute() external onlyOwner {
```

Comment:
This function can be `onlyManager` instead so it doesn't need to go through the multi-sig.